gettsim/renten_anspr.py

```diff
@@ -0,0 +1,139 @@
+def rente_anspr_m(zugangsfaktor, entgeltpunkte_update, rentenwert):
+    """ This function calculates the Old-Age Pensions claim if the agent chooses to
+    retire. The function basically follows the following equation:
+
+    .. math::
+
+        R = EP * ZF * Rw
+
+    models 'Rentenformel':
+    https://de.wikipedia.org/wiki/Rentenformel
+    https://de.wikipedia.org/wiki/Rentenanpassungsformel
+
+    - In particular, it calculates the "entgeltpunkte" for the previous year, based on
+      earnings of that year. These need to be related to average earnings
+    - As we do not know previously collect Entgeltpunkte, we take an average
+      value (to be improved)
+
+    Parameters
+    ----------
+    zugangsfaktor
+    entgeltpunkte_update
+    ges_renten_vers_params
+
+    Returns
+    -------
+
+    """
+
+    out = (entgeltpunkte_update * zugangsfaktor * rentenwert).clip(lower=0)
```

Comment:
The statement can be returned directly - no need to define out. (similar for the other functions here)

Reply:
Well, that's actually a discussion, we had quite often and didn't come yet to a conclusion. We said before to avoid complicated calculations at the return statement for readability reasons. Here it is not complicated, but I find it more readable. Can change if you like!